tests/torch/nas/test_flops.py

```diff
@@ -50,6 +50,26 @@
         ),
         blocks_to_skip=RESNET50_BLOCK_TO_SKIP,
     ),
+    MultiElasticityTestDesc(
+        name='resnet50_width_overwrite_groups',
+        model_creator=resnet50_cifar10,
+        ref_model_stats=RefModelStats(
+            supernet=ModelStats(651_599_872, 23_467_712),
+            kernel_stage=ModelStats(651_599_872, 23_467_712),
+            depth_stage=ModelStats(615_948_288, 23_398_080),
+            width_stage=ModelStats(589_209_600, 23_238_336)
+        ),
+        blocks_to_skip=RESNET50_BLOCK_TO_SKIP,
+        algo_params={'width': {
+            "overwrite_groups": [
```

Comment:
Why do you need to overwrite groups? I doubt that eventually user will use it.

Reply:
@ljaljushkin  Overwrite groups allow the user to have a more granular control over the levels of elastic width that might be applied. It has proven to be a very useful knob. Thanks!

Reply:
But it would mean no automation for elastic width, right?

Reply:
@ljaljushkin No, automation is still important. For instance, we first try with elasticity recommended by NNCF. However, it is important to also allow the user to go beyond the automated recommendation. Thanks!